cmd/webhook/main.go

```diff
@@ -37,6 +41,22 @@ import (
 	"knative.dev/pkg/webhook"
 )
 
+const (
+	RegistrationDelayTime = "REG_DELAY_TIME"
+)
+
+func getRegistrationDelayTime() time.Duration {
+	var RegistrationDelay time.Duration
+	if rdt := os.Getenv(RegistrationDelayTime); rdt != "" {
```

Comment:
it would be better to migrate to `envconf`.

Reply:
Done